test_coverage_summary.py

```
@@ -0,0 +1,215 @@
+            
+        try:
+            with open(src_file, 'r') as f:
+                content = f.read()
+            tree = ast.parse(content)
+            
+            functions = []
+            classes = []
+            mcp_tools = []
+            
+            # Simple approach: look for @mcp.tool() in content
+            mcp_tool_lines = [line for line in content.split('\n') if '@mcp.tool' in line]
```

Ruff (0.12.2)

Local variable `mcp_tool_lines` is assigned to but never used

Remove assignment to unused variable `mcp_tool_lines`

(F841)